packages/fx-core/src/plugins/resource/apiconnector/index.ts

```diff
@@ -40,6 +41,19 @@ export class ApiConnectorPluginV2 implements ResourcePlugin {
     envInfo: DeepReadonly<v2.EnvInfoV2>,
     tokenProvider: TokenProvider
   ): Promise<ApiConnectorResult> {
+    const res = await ctx.userInteraction?.showMessage(
+      "warn",
+      getLocalizedString("plugins.apiconnector.ExecuteUserTask.Message"),
+      true,
+      "OK"
+    );
+    const answer = res?.isOk() ? res.value : undefined;
+    if (!answer || answer != "OK") {
+      throw ResultFactory.UserError(
```

Comment:
Will this error be displayed to user? Since the cancel button is clicked by user, we may not want to show cancel error to user again. But let's ensure we have telemetry to record how many users clicked cancel.

Reply:
this will display in the output, like cancel any other command pallette with clicking Esc